src/cdk-experimental/drag-drop/drag-drop-registry.ts

```diff
@@ -20,35 +18,35 @@ const activeEventOptions = supportsPassiveEventListeners() ? {passive: false} :
 type PointerEventHandler = (event: TouchEvent | MouseEvent) => void;
 
 /**
- * Service that keeps track of all the `CdkDrag` and `CdkDrop` instances, and
- * manages global event listeners on the `document`.
+ * Service that keeps track of all the drag item and drop container
+ * instances, and manages global event listeners on the `document`.
  * @docs-private
  */
 @Injectable({providedIn: 'root'})
-export class CdkDragDropRegistry implements OnDestroy {
+export class CdkDragDropRegistry<I, C extends {id: string}> implements OnDestroy {
```

Comment:
Now that the things being dragged and dropped are generic, just make the class `DragDropRegistry`? The general rule is that things only start with `Cdk` if they're a directive with the `cdk-` selector prefix, but I interpreted the class name before as "registry for CdkDrag and CdkDrop". Since they're generics now, I don't think we should have the prefix